Demo/src/androidTest/java/com/braintreepayments/demo/test/DropInTest.java

```diff
@@ -163,7 +166,7 @@ public void tokenizesPayPal() {
         onDevice(withText("Add Payment Method")).waitForExists().waitForEnabled().perform(click());
 
         onDevice(withText("PayPal")).perform(click());
-        onDevice(withContentDescription("Proceed with Sandbox Purchase")).perform(click());
+        onDevice(withTextStartingWith("Proceed with Sandbox")).perform(click());
```

Comment:
This works. Although for more robustness we could use `withText("Proceed with Sandbox Purchase")`

There are 2 other examples further down that need this change.